src/ecs-mcp-server/awslabs/ecs_mcp_server/modules/troubleshooting.py

```diff
@@ -79,10 +78,12 @@ async def mcp_ecs_troubleshooting_tool(
 
         ### 1. get_ecs_troubleshooting_guidance
         Initial assessment and data collection
-        - Required: app_name
-        - Optional: symptoms_description (Description of symptoms experienced by the user)
+        - Required: cluster_name
+        - Optional: service_name (Name of the ECS service to troubleshoot),
```

Comment:
where do we provide region? how does mcp know where to troubleshoot?

Reply:
It troubleshoots the account and region defined in the MCP config. Can create backlog item to troubleshoot any region in the AWS account with optional param.